options.go

```diff
@@ -244,7 +243,7 @@ func (opts *options) handleFifos() (io.Writer, error) {
 	}
 
 	if generateFifoFilename || generateMetricFifoFilename {
-		dir, err := ioutil.TempDir(os.TempDir(), "fcfifo")
+		dir, err := os.MkdirTemp(os.TempDir(), "fcfifo")
```

Comment:
An empty string as the first parameter will choose this TempDir as the default, iiuc

Reply:
Good call out. Might as well be consistent. :)